flake.nix

```diff
@@ -25,6 +25,11 @@
           inherit system;
           overlays = [
             self.overlays.default
+            (final: prev: {
+              openssl_1_1 = prev.openssl_1_1.overrideAttrs (old: {
+                meta = [];
```

Comment:
Maybe better move it inside the default overlay so that it still works when people use the `overlay` output instead of `packages`.